### DISCUSSION TITLE
nuclei context deadline exceeded

### DISCUSSION THREAD

**Discussion by mnd5756242** [projectdiscovery/nuclei#4535] [Q&A]
I called the following method to execute ：
        e := poc.Executer
	ctx := scan.NewScanContext(contextargs.NewWithInput(target))
	results, err = e.ExecuteWithResults(ctx)    
reported an error：context deadline exceeded
What is the reason

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
